docs/requirements-docs.txt

```diff
@@ -1,7 +1,8 @@
 numpydoc
 sphinx>=4.0.0
-dask-sphinx-theme>=1.3.5
+git+https://github.com/jsignell/dask-sphinx-theme@docs-refresh
 sphinx-click
+sphinx-tabs
 toolz
 cloudpickle
 pandas>=0.25.0
```

Comment:
In this file we are installing distributed but the line `git+https://github.com/dask/distributed` seems to only work with pip similarly to the new line added `git+https://github.com/jsignell/dask-sphinx-theme@docs-refresh`
See https://github.com/dask/dask/issues/8188 for reference. 
Should we remove the `conda` installation instructions or is there a workaround so the file `requirements-doc.txt`can work both with `pip` and `conda`

Reply:
Good point! I guess it's been a while since I removed and rebuilt the docs env.

Reply:
Ok I changed the recommendation to only use pip.